configure-secrets.sh

```diff
@@ -40,8 +48,36 @@ echo "//" >> $SECRETS_FILE
 echo "// Add new secrets configuration in ./configure-secrets.sh" >> $SECRETS_FILE
 echo "//" >> $SECRETS_FILE
 echo "// In order to have the // in https://, we need to split it with an empty" >> $SECRETS_FILE
-echo "// variable substitution via \$() e.g. ROOT_URL = https:/\$()/www.endpoint.com" >> $SECRETS_FILE
+echo "// variable substitution via \$() e.g. ROOT_URL = https:/\$()/www.example.com" >> $SECRETS_FILE
+
+longest_name=-1
+for var_name in ${vars[@]}
+do
+	current=${#var_name}
+   if [ $current -gt $longest_name ]
+   then
+      longest_name=$current
+   fi
+done
+
+success=true
+for var_name in ${vars[@]}
+do
+  nbstars=$(($longest_name - ${#var_name} + 5))
+  stars=$(printf '%*s' $nbstars '')	
+  value=$(eval "echo \$$var_name")
+  if [ -z "$value" ]; then
+    >&2 echo "➜ ${BOLD}$var_name${NOCOLOR} ${stars// /*} ${RED}ERROR!${NOCOLOR} Missing environment variable ${BOLD}$var_name${NOCOLOR}."
+    success=false
+  else
+  	echo "➜ ${BOLD}$var_name${NOCOLOR} ${stars// /*} ${GREEN}OK${NOCOLOR}"
+  fi
+  secureDoubleSlashForXCConfig $value
+  # Set secrets from environment variables
+  cmd='echo "${var_name} = $SECURE_DOUBLE_SLASH_FOR_XCCONFIG_RESULT"'
+  eval ${cmd} >> $SECRETS_FILE
+done
 
-# Set secrets from environment variables
-echo "RIAM_CONFIG_URL = $CONFIG_URL" >> $SECRETS_FILE
-echo "RIAM_APP_SUBSCRIPTION_KEY = $SUBSCRIPTION_KEY" >> $SECRETS_FILE
\ No newline at end of file
+if [ $success = false ] ; then
+	echo "Before building the project you must set missing environment variables. See project README for instructions."
```

Comment:
@Climbatize 👍 
I think the README might need updated too.

Reply:
But there are already some instructions in README to set up env variables for Test app.